<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>WitdrawalWindow</class>
 <widget class="QDialog" name="WitdrawalWindow">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>400</width>
    <height>309</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>E-Bankomat</string>
  </property>
  <widget class="QWidget" name="layoutWidget">
   <property name="geometry">
    <rect>
     <x>70</x>
     <y>20</y>
     <width>258</width>
     <height>269</height>
    </rect>
   </property>
   <layout class="QVBoxLayout" name="verticalLayout">
    <item>
     <widget class="QLabel" name="atmLabel">
      <property name="font">
       <font>
        <family>Bahnschrift SemiLight SemiConde</family>
        <pointsize>25</pointsize>
       </font>
      </property>
      <property name="text">
       <string>Bankomat</string>
      </property>
      <property name="alignment">
       <set>Qt::AlignCenter</set>
      </property>
     </widget>
    </item>
    <item>
     <layout class="QHBoxLayout" name="horizontalLayout_3">
      <item>
       <widget class="QLabel" name="accountLabel">
        <property name="font">
         <font>
          <pointsize>12</pointsize>
         </font>
        </property>
        <property name="text">
         <string>Konto</string>
        </property>
       </widget>
      </item>
      <item>
       <widget class="QComboBox" name="accountSelectionBox"/>
      </item>
     </layout>
    </item>
    <item>
     <layout class="QHBoxLayout" name="horizontalLayout">
      <item>
       <widget class="QLabel" name="amountLabel">
        <property name="font">
         <font>
          <pointsize>13</pointsize>
         </font>
        </property>
        <property name="text">
         <string>Kwota</string>
        </property>
       </widget>
      </item>
      <item>
       <widget class="QLineEdit" name="ammountInput"/>
      </item>
     </layout>
    </item>
    <item>
     <layout class="QHBoxLayout" name="horizontalLayout_4">
      <item>
       <widget class="QLabel" name="balanceLabel">
        <property name="font">
         <font>
          <pointsize>13</pointsize>
         </font>
        </property>
        <property name="text">
         <string>Saldo:</string>
        </property>
       </widget>
      </item>
      <item>
       <widget class="QLineEdit" name="balanceInput"/>
      </item>
     </layout>
    </item>
    <item>
     <layout class="QHBoxLayout" name="horizontalLayout_2">
      <item>
       <widget class="QPushButton" name="depositButton">
        <property name="text">
         <string>Wpłać</string>
        </property>
       </widget>
      </item>
      <item>
       <widget class="QPushButton" name="withdrawButton">
        <property name="text">
         <string>Wypłać</string>
        </property>
       </widget>
      </item>
     </layout>
    </item>
    <item>
     <widget class="QPushButton" name="randAmountButton">
      <property name="text">
       <string>Losuj</string>
      </property>
     </widget>
    </item>
    <item>
     <widget class="QPushButton" name="exitButton">
      <property name="text">
       <string>Wróć</string>
      </property>
     </widget>
    </item>
   </layout>
  </widget>
 </widget>
 <resources/>
 <connections/>
</ui>
